packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -0,0 +1,90 @@
+import { RadioProps, RadioState } from './Radio.types';
+import * as React from 'react';
+import { useRadioGroupContext } from '../RadioGroup/radioGroupContext';
+import { useAsPressable, useOnPressWithFocus, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+
+export const radioSelectActionLabel = 'Select a RadioButton';
```

Comment:
>'Select a RadioButton' [](http://example.com/codeflow?start=38&length=22)

We shouldn't use hardcoded strings as labels/accessibility labels

Reply:
Removed all instances of hard-coded strings, thanks!